debian/precise/foreman/control

```diff
@@ -88,16 +88,25 @@ Package: foreman-compute
 Architecture: all
 Section: web
 Priority: extra
-Replaces: foreman-fog
-Provides: foreman-fog
-Conflicts: foreman-fog
+Recommends: foreman-ec2
 Depends: foreman, libxml2-dev, libxslt1-dev, ${misc:Depends}
-Description: metapackage providing fog dependencies for Foreman (for Amazon EC2 support)
```

Comment:
The RPMs say it includes OpenStack and Rackspace support, perhaps say that instead?